__tests__/core/runner.test.ts

```diff
@@ -147,6 +147,21 @@ describe('runner', () => {
     });
   });
 
+  it('run journey - failed on beforeAll', async () => {
+    const error = new Error('Broken beforeAll hook');
+    runner.addHook('beforeAll', () => {
+      throw error;
+    });
+    runner.addJourney(new Journey({ name: 'j1' }, () => step('step1', noop)));
+    const result = await runner.run({
+      wsEndpoint,
+      outfd: fs.openSync(dest, 'w'),
+    });
+    expect(result).toEqual({
```

Comment:
Can we add a second journey to this test? The hook should cause all journeys to fail, so we should assert it for > 1 journey

Reply:
Yes good point, will add it.